profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.admin.inc

```diff
@@ -24,6 +24,67 @@ function nexteuropa_varnish_admin_settings_form($form, $form_state) {
     '#default_value' => variable_get('nexteuropa_varnish_default_purge_rule', FALSE),
   );
 
+  $form = system_settings_form($form);
+  array_unshift($form['#submit'], '_nexteuropa_varnish_default_rule_state_check');
```

Comment:
Why must it be executed before the actual saving of the settings? And why do we need another message; the default message "The configuration option have been saved" is not enough?
There is only one option; its checkbox and its label are obvious enough to avoid an additional step in the form submit process, no?